content/03.use-cases-a.md

```diff
@@ -34,16 +25,16 @@ Some EEB labs have even turned their lab notebooks into shareable websites [@url
 GitHub supports a broad set of mechanisms for hosting educational materials.
```

Comment:
I know a few classes that have used GitHub (or GitHub pages hosted websites) for their courses rather than using their University's course management system (e.g. Blackboard or Canvas), but they are all very computer savvy or are teaching programming-centric courses. I think using GitHub for course management is a tough sell, so maybe this section can be more clear in the first paragraph about what situations make GitHub a worthwhile solution (e.g. you don't have access to Canvas, you need more customizability, a goal of your class is for students to become familiar with GitHub, you hate Canvas enough to just build a whole website on your own instead, you want your course materials to be open access, etc.)

Reply:
That makes sense @Aariq. And I think we do mention just a few of those reasons for using github (like that material is open access rather than behind a canvas login).  

One way to potentially modify this section then, is to state upfront the reasons Eric mentioned why teachers might put in **extra** effort to run their course through GitHub.  Also could potentially reference our discussion where we specifically call out that GitHub is never formally taught in school, but rather an 'add on' to programming courses.  I'm not saying that this entirely solves it, but it's at least another modality for using GitHub.